tests/Builder/FormContractorTest.php

```diff
@@ -84,7 +83,8 @@ public function testDefaultOptionsForSonataFormTypes(): void
             AdminType::class,
         ];
         $collectionTypes = [
-            DeprecatedCollectionType::class,
+            // NEXT_MAJOR: 'Sonata\CoreBundle\Form\Type\CollectionType'
+            'Sonata\CoreBundle\Form\Type\CollectionType',
```

Comment:
On the OrmAdminBundle this line was removed to not break tests

Reply:
Thanks!